package.json

```diff
@@ -10,26 +10,9 @@
     "type": "git",
     "url": "https://github.com/MetaMask/test-dapp-multichain.git"
   },
-  "sideEffects": false,
-  "type": "module",
-  "exports": {
-    ".": {
-      "import": {
-        "types": "./dist/index.d.mts",
-        "default": "./dist/index.mjs"
-      },
-      "require": {
-        "types": "./dist/index.d.cts",
-        "default": "./dist/index.cjs"
-      }
-    },
-    "./package.json": "./package.json"
-  },
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.cts",
+  "main": "index.js",
```

Comment:
Might be easier to read if we include this in `files`? But apparently it will be included by virtue of it being the `main` field, so this should work as-is